### ISSUE TITLE
Warn user that they are working with one or more file name(s) that already exist on Wikimedia Commons

### ISSUE THREAD

**Issue by trnstlntk** [OpenRefine/CommonsExtension#32]
File names on Wikimedia Commons must be unique (two files can't have the same name).

The default Wikimedia Commons UploadWizard warns the user when they are naming a file the same as one that already exists.

![Image](https://user-images.githubusercontent.com/9881645/166155708-0f02c197-56fa-40ca-970c-1f1c30221562.png)

It would be great if OpenRefine also warns uploaders of new Commons files if this happens. I can imagine this will be part of the 'Issues' tab when creating a schema for uploading files to Commons, see #22

- [ ] @lozanaross to design a simple 'warning mechanism'
- [ ] @wetneb or @j-sal to build the actual checking mechanism

We didn't put a deadline for this on the post it when doing our big planning exercise last week. I'd say June 30 just like #22 (this is actually part of that task)

**Comment by lozanaross** [comment]
Assigning this to myself so I can filter for it :-) Also related to #33.

**Comment by trnstlntk** [comment]
Doing some end-of-project cleanup. I think we never got around to doing this one, although it would be useful. @lozanaross I feel inclined to unassign this issue from you and generalize its description so that someone else can get to it at a later point (both design and development-wise). Agreed?

**Comment by trnstlntk** [comment]
It's probably also relevant for Wikibase uploads, as duplicate file names are not allowed there either?

**Comment by lozanaross** [comment]
Yes, all fine to generalize issue. I have some ideas related to upload process and error logging that I'll try to do a final design on as part of the report wrap-up of the final sessions, but that won't be something necessary to be built or finalized in the short term.
